tests/installation/boot_into_snapshot.pm

```diff
@@ -20,9 +20,12 @@ use strict;
 use warnings;
 use testapi;
 use base "opensusebasetest";
+use bootloader_setup qw(stop_grub_timeout boot_into_snapshot);
 
 sub run {
     my ($self) = @_;
+    stop_grub_timeout;
```

Comment:
We have more steps in `grub_test` which we should have here. Like entering password if boot partition is encrypted and asserting that grub is shown. And some special steps for xen, for which we don't have VR.

TBH, now I'm not sure if this separation will make out lives any better....